packages/shared/components/modals/AccountActionsMenu.svelte

```diff
@@ -9,7 +9,8 @@
 
     export let modal: Modal
 
-    const shouldDisableDelete = $selectedAccount.meta.index !== $activeAccounts?.length - 1
+    const hideDelete =
```

Comment:
Is this something that should be changed in this PR? Was there a bug around this code?

Reply:
This PR should close #4462

Reply:
I would move this update to a different PR to leave this PR as a refactor only one 🙏🏼